plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -633,10 +634,17 @@ class DefaultPsiToDocumentableTranslator(
             else -> getBound(type)
         }
 
-        private fun PsiModifierListOwner.getModifier() = when {
-            hasModifier(JvmModifier.ABSTRACT) -> JavaModifier.Abstract
-            hasModifier(JvmModifier.FINAL) -> JavaModifier.Final
-            else -> JavaModifier.Empty
+        private fun PsiModifierListOwner.getModifier(): JavaModifier {
+            val isInterface = this is PsiClass && this.isInterface
+            if (isInterface) {
+                return JavaModifier.Empty // all Java modifiers are redundant for interfaces
```

Comment:
Nitpicking: The comment is a little confusing. Interfaces can have an access modifier.